test/PizzeriaTest.cpp

```diff
@@ -65,3 +83,38 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, completeOrderWith3Pizzas)
+{
+    PizzaMock* mockMargerita = new StrictMock<PizzaMock>{};
+    ON_CALL(*mockMargerita, getPrice()).WillByDefault(Return(11.11));
+    ON_CALL(*mockMargerita, getBakingTime()).WillByDefault(Return(minutes(10)));
+    ON_CALL(*mockMargerita, getName()).WillByDefault(Return("Margerita"));
+
+    PizzaMock* mockFungi = new NiceMock<PizzaMock>{};
+    ON_CALL(*mockFungi, getPrice()).WillByDefault(Return(22.22));
+    ON_CALL(*mockFungi, getBakingTime()).WillByDefault(Return(minutes(90)));
+
+    PizzaStub* pizzaStub = new PizzaStub{"STUB"};
+
+    Pizzas pizzas = {mockFungi,mockMargerita,pizzaStub};
+
+    EXPECT_CALL(*m_timerMock_rawPtr , sleep_for(_)).Times(3);
```

Comment:
_ take anthing,  better put here time which you to calls :)  Also time and price are magic values

Reply:
Exactly. We do care in this test for how long sleep_for was called. It should be a baking time of each pizza. We need 3 separate expectations on them. It enforces you to use constexpr values instead of magic ones.